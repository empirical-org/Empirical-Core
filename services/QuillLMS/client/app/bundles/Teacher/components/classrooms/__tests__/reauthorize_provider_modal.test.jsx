import { shallow } from 'enzyme';
import React from 'react';

import ReauthorizeProviderModal from '../reauthorize_provider_modal';

describe('ReauthorizeProviderModal component', () => {
  const close = () => {}

  const wrapper = shallow(
    <ReauthorizeProviderModal
      close={close}
      link=''
      provider='Google'
    />
  );

  it('should render ReauthorizeProviderModal', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
