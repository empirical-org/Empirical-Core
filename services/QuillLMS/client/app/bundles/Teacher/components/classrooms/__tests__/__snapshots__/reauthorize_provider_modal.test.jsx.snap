// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReauthorizeProviderModal component should render ReauthorizeProviderModal 1`] = `
<div
  className="modal-container reauthorize-provider-modal-container"
>
  <div
    className="modal-background"
  />
  <div
    className="reauthorize-provider-modal quill-modal modal-body"
  >
    <div>
      <h3
        className="title"
      >
        Reauthorize
        Google
      </h3>
    </div>
    <p>
      To import a new
      Google
       classroom, you need to reauthorize
      Google
       access.
      <br />
      Clicking reauthorize will re-direct you to
      Google
      .
    </p>
    <div
      className="form-buttons"
    >
      <button
        className="quill-button-archived focus-on-light outlined secondary medium"
        onClick={[Function]}
        type="button"
      >
        Cancel
      </button>
      <AuthGoogleAccessForm
        buttonClass="quill-button-archived focus-on-light contained primary medium"
        text="Reauthorize"
      />
    </div>
  </div>
</div>
`;
